### DISCUSSION TITLE
Mount a custom Event Listener in Quarkus container

### DISCUSSION THREAD

**Discussion by PaulBarrie** [keycloak/keycloak#12515] [Q&A] [answered]
Hi, 

Moving from the jboss distro to the quarkus`one I need to mount a custom event listener.

With jboss distribution I just had to mount my jar `/opt/jboss/keycloak/standalone/deployments/` but I can't figure out where I should put my jar with the quarkus distro :/

Thanks for your answers :)

**Comment by dasniko** [comment, marked as answer]
Read the docs: https://www.keycloak.org/server/configuration-provider#_installing_and_uninstalling_a_provider
`/opt/keycloak/providers`

**Reply by PaulBarrie** [reply]
Thanks :)
